<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modi</class>
 <widget class="QDialog" name="modi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>449</width>
     <height>186</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Padauk</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Hora</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="hora_3">
          <property name="accelerated">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string/>
          </property>
          <property name="prefix">
           <string/>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>12</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Padauk</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Minutos</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="min_3">
          <property name="maximum">
           <number>59</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QComboBox" name="universal">
          <property name="toolTip">
           <string/>
          </property>
          <property name="accessibleDescription">
           <string/>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="currentText">
           <string>AM</string>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <property name="maxVisibleItems">
           <number>3</number>
          </property>
          <property name="maxCount">
           <number>3</number>
          </property>
          <property name="insertPolicy">
           <enum>QComboBox::InsertAtCurrent</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
          </property>
          <item>
           <property name="text">
            <string>AM</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PM</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>M</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="actividad_2">
      <property name="placeholderText">
       <string>Ingrese Actividad</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="TIPO">
      <item>
       <property name="text">
        <string>Acad√©micas</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rutinarias</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Eventuales</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="lun">
        <property name="font">
         <font>
          <family>Padauk</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Lun</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mar">
        <property name="font">
         <font>
          <family>Padauk</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Mar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mier">
        <property name="font">
         <font>
          <family>Padauk</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Mie</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="jue">
        <property name="font">
         <font>
          <family>Padauk</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Jue</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="vie">
        <property name="font">
         <font>
          <family>Padauk</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Vie</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="sab">
        <property name="font">
         <font>
          <family>Padauk</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Sab</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="dom">
        <property name="font">
         <font>
          <family>Padauk</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Dom</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Padauk</family>
     <pointsize>28</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Modificar</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>360</y>
     <width>451</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QCommandLinkButton" name="AGREGAR">
      <property name="font">
       <font>
        <family>Padauk</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Modificar</string>
      </property>
      <property name="icon">
       <iconset resource="icono.qrc">
        <normaloff>:/new/prefix1/gua.png</normaloff>:/new/prefix1/gua.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QCommandLinkButton" name="REGRESAR">
      <property name="font">
       <font>
        <family>Padauk</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Regresar</string>
      </property>
      <property name="icon">
       <iconset resource="icono.qrc">
        <normaloff>:/new/prefix1/regre.png</normaloff>:/new/prefix1/regre.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="icono.qrc"/>
 </resources>
 <connections/>
</ui>
